apps/cow-fi/components/AddRpcButton/index.tsx

```
@@ -1,4 +1,4 @@
-import { Confetti } from '@cowprotocol/ui'
+import { Confetti, Color } from '@cowprotocol/ui'
```

ESLint

[error] There should be at least one empty line between import groups

(import/order)